### PR TITLE
ccw: Enable DASD device selection via configuration

### PR DESCRIPTION
**Pull request by smitterl**
Depends on https://github.com/avocado-framework/avocado-vt/pull/3606

Tests relied on a function in avocado-vt to select a device for testing based on certain assumptions.

Add a parameter to the configurations in order to overwrite that selection and choose a device manually.

Signed-off-by: Sebastian Mitterle <smitterl@redhat.com>

### CODE REVIEW

**Comment by smitterl** [comment]
```
# avocado run --vt-type libvirt --vt-machine-type s390-virtio --vt-connect-uri qemu:///system libvirt_ccw_passthrough --vt-extra-params "devid = 0.0.5300" --test-runner=runner
JOB ID     : 920666eb0aa64f69dad4b980f2484c7f50ae5ab7
JOB LOG    : /var/lib/avocado/job-results/job-2023-01-20T12.14-920666e/job.log
 (1/3) type_specific.io-github-autotest-libvirt.libvirt_ccw_passthrough.read_write.happy_path: PASS (50.86 s)
 (2/3) type_specific.io-github-autotest-libvirt.libvirt_ccw_passthrough.happy_path: PASS (23.74 s)
 (3/3) type_specific.io-github-autotest-libvirt.libvirt_ccw_passthrough.device_removal: PASS (35.98 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2023-01-20T12.14-920666e/results.html
JOB TIME   : 110.95 s
# avocado run --vt-type libvirt --vt-machine-type s390-virtio --vt-connect-uri qemu:///system virtual_disks.dasd --vt-extra-params "devid = 0.0.5300" --test-runner=runner
JOB ID     : bdb95cd2134617301dce919a51d40e806c30db5e
JOB LOG    : /var/lib/avocado/job-results/job-2023-01-20T12.16-bdb95cd/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virtual_disks.dasd.read_native_partition_table: PASS (12.07 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2023-01-20T12.16-bdb95cd/results.html
JOB TIME   : 12.49 s
```

File: libvirt/tests/cfg/passthrough/ccw/libvirt_ccw_passthrough.cfg [hunk 1]
```diff
@@ -1,6 +1,7 @@
 - libvirt_ccw_passthrough:
     type = libvirt_ccw_passthrough
     only s390-virtio
+    devid =
```
Comment (chunfuwen) [inline libvirt/tests/cfg/passthrough/ccw/libvirt_ccw_passthrough.cfg:4]: I assume this value need be pre-configured in libvirt CI to make test case pass. 
So devid = "EXAMPLE_DEVICE_ID_NEED_REPLACED" could be more explicitly expressed

Comment (smitterl) [reply libvirt/tests/cfg/passthrough/ccw/libvirt_ccw_passthrough.cfg:4]: Yes but in this case I didn't want to set a default value just in case because the test case could be harmful.
If I change the code as you suggest, I would need to add logic in the test script to set "" or None as value if this string isn't set. Therefore, I prefer to keep it as it is.

**Comment by smitterl** [comment]
@dzhengfy Please can you help review?

**Review by chloerh — APPROVED** [review]

**Review by chunfuwen — APPROVED** [review]
LGTM
